packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -11,13 +11,17 @@ import { version } from '../../../package.json';
 
 const d = debug('electron-forge:init:npm');
 
+function siblingDep(name: string) {
+  return `@electron-forge/${name}@${version}`;
```

Comment:
I hope it's sweet enough!

Reply:
Oh, I see I missed the section about `npm run commit` in contributing notice. I'm sorry for the inconvenience.